packages/govern-server/src/core/data/TheGraphClient.ts

```diff
@@ -0,0 +1,71 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import { ErrorConnection } from '../errors'
+import { QueryResult } from '../types'
+
+type TheGraphClientOptions = {
+  verbose?: boolean
+}
+
+class TheGraphClient {
+  #client: Client
+  #verbose: boolean
+
+  constructor(
+    subgraphUrl: string,
+    options: TheGraphClientOptions | boolean = {}
+  ) {
+    this.#verbose = this.parseOptions(options).verbose ?? false
+    this.#client = new Client({ maskTypename: true, url: subgraphUrl, fetch })
+  }
+
+  private parseOptions(
```

Comment:
It isn't required to have it on a new line here cause it does not exceed the line-length.